mne/time_frequency/multitaper.py

```diff
@@ -366,7 +366,8 @@ def _compute_mt_params(n_times, sfreq, bandwidth, low_bias, adaptive,
 @verbose
 def psd_array_multitaper(x, sfreq, fmin=0.0, fmax=np.inf, bandwidth=None,
                          adaptive=False, low_bias=True, normalization='length',
-                         output='power', n_jobs=None, verbose=None):
+                         output='power', n_jobs=None, max_iter=150, *,
```

Comment:
This would be the correct order to ensure accidental verbose `arg` placement doesn't mess up people's scripts
```suggestion
                         output='power', n_jobs=None, *, max_iter=150,
```